types/react-beautiful-dnd/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-beautiful-dnd 13.0
+// Type definitions for react-beautiful-dnd 13.1
 // Project: https://github.com/atlassian/react-beautiful-dnd
 // Definitions by: varHarrie <https://github.com/varHarrie>
```

Comment:
@brianspowers could you add yourself to the list of authors to get tagged for feature prs?